airflow/www/static/js/trigger.js

```diff
@@ -147,18 +147,18 @@ function initForm() {
       });
     });
 
-    // Ensure layout is refreshed on generated JSON as well
-    document.getElementById('generated_json_toggle').addEventListener('click', () => {
-      setTimeout(jsonForm.refresh, 300);
-    });
+    // // Ensure layout is refreshed on generated JSON as well
```

Comment:
Why did you disable this? Was there an error in testing?
This is needed because the section is hidden during load of the form and then the JSON view is not correctly initialized - don't know why - workaround was to initialize it lazy by this.